itests/src/test/resources/testconfiguration.properties

```diff
@@ -241,7 +241,6 @@ mr.query.files=\
   join_vc.q,\
   list_bucket_dml_8.q,\
   localtimezone.q,\
-  manyViewJoin.q,\
```

Comment:
Why can we test HIVE-28326 using `manyViewJoin.q`? I applied the change of `StageIDsRearranger.java`, built it, and run `mvn test -Dtest=TestCliDriver -Dtest.output.overwrite=false -Dqfile=manyViewJoin.q`. The test succeeded on my machine.

Reply:
I understand the test case with the original revision fails with NPE.

```
See ./ql/target/tmp/log/hive.log or ./itests/qtest/target/tmp/log/hive.log, or check ./ql/target/surefire-reports or ./itests/qtest/target/surefire-reports/ for specific test cases logs.
 java.lang.NullPointerException
	at org.apache.hadoop.hive.ql.optimizer.physical.LlapDecider.resolve(LlapDecider.java:530)
	at org.apache.hadoop.hive.ql.parse.TezCompiler.optimizeTaskPlan(TezCompiler.java:834)
	at org.apache.hadoop.hive.ql.parse.TaskCompiler.compile(TaskCompiler.java:340)
```